Make which role is needed for mongo more clear. (#48)
ni/systemlink-operations-handbook: handbook/data-stores/mongodb.md

- A server running SystemLink 2021 R1 or greater. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account

> - A user with the `readWriteAnyDatabase` role. The user must have the `createCollection` privilege and be able to create new databases. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. .

Reviewer: ```suggestion
- A user with the `readWriteAnyDatabase` role. The user must have the `createCollection` privilege and be able to create new databases. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details.
```

- A server running SystemLink 2021 R1 or greater. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account

> - A user with the `readWriteAnyDatabase` role. This is NOT the same as a user with the `readWrite` role because the user will need the `createCollection` privilege for new top level databases, not just creating collections within existing databases. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. You reference this user in the MongoDB connection fields or connection string in **NI SystemLink Server Configuration**.

Reviewer: ```suggestion
- A user with the `readWriteAnyDatabase` role. The user must have the `createCollection` privilege and be able to create new databases. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. .
```

> - A user with the `readWriteAnyDatabase` role. This is NOT the same as a user with the `readWrite` role because the user will need the `createCollection` privilege for new top level databases, not just creating collections within existing databases. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details. You reference this user in the MongoDB connection fields or connection string in **NI SystemLink Server Configuration**.

Reviewer: Should the new note be in an "important" section?

Author: I made it more terse and moved it to an important section.
